### PR TITLE
Add io.github.kaii_lb.Overskride

### PR DESCRIPTION
**Pull request by kaii-lb**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

For permissions, app needs: 
- both session and system dbus access for bluetooth to work (dbus-rs libraries).
- filesystem access to send and receive files

### CODE REVIEW

File: io.github.kaii_lb.Overskride.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    ],
+    "command" : "overskride",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--allow=bluetooth",
+        "--socket=system-bus",
+        "--socket=session-bus",
```
Comment (bbhtt) [inline io.github.kaii_lb.Overskride.json:17]: This isn't allowed by default and I doubt it is correct.

Please try figure out which specific bus names are needed using --log-session-bus, dbus monitor etc.

File: io.github.kaii_lb.Overskride.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        "--filesystem=xdg-run/gvfsd",
+        "--talk-name=org.gtk.vfs.*"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```
Comment (bbhtt) [inline io.github.kaii_lb.Overskride.json:24]: Not allowed, build is done offline

File: io.github.kaii_lb.Overskride.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        ],
+        "env" : {
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "overskride=debug",
+            "CARGO_HOME" : "/run/build/hgui-rust/cargo",
```
Comment (bbhtt) [inline io.github.kaii_lb.Overskride.json:29]: Doesn't seem correct, there is no module called hgui-rust

File: io.github.kaii_lb.Overskride.json (hunk 1 above)
Comment (hfiguiere) [inline io.github.kaii_lb.Overskride.json:17]: this is not allowed. You need explicit names specified.

File: io.github.kaii_lb.Overskride.json (hunk 3 above)
Comment (hfiguiere) [inline io.github.kaii_lb.Overskride.json:29]: this is incorrect. the path depends on the name of the module.

File: io.github.kaii_lb.Overskride.json (hunk 1 above)
Comment (bbhtt) [reply io.github.kaii_lb.Overskride.json:17]: Please try figure out which specific bus names are needed using --log-session-bus, dbus monitor etc.

File: io.github.kaii_lb.Overskride.json (hunk 2 above)
Comment (hfiguiere) [inline io.github.kaii_lb.Overskride.json:24]: this is not allowed. build must be performed off-line

**Pull request closed without merging**